kqueen/blueprints/api/views.py

```diff
@@ -220,6 +220,29 @@ def cluster_resize(pk):
     return jsonify(output)
 
 
+@api.route('/clusters/<uuid:pk>/set_network_policy', methods=['PATCH'])
+@jwt_required()
+def cluster_set_network_policy(pk):
+    obj = get_object(Cluster, pk, current_identity)
+
+    data = request.json
+    if not isinstance(data, dict) or (isinstance(data, dict) and 'network_policy' not in data):
```

Comment:
Please take a look at my previous comments

Reply:
Please, add a separate check for application/json above and remove isinstance check at all

if not request.json:
        abort(400, description='JSON data expected')